Typescript 3.1 - compile error. toggleAttribute type conflict in  LegacyElementMixin

<!-- Instructions For Filing a Bug: https://github.com/Polymer/polymer/blob/master/CONTRIBUTING.md#filing-bugs -->
### Description

The following error is seen in typescript 3.1 when using any element with the LegacyElementMixin:  

` error TS2320: Interface 'AppDrawerElement' cannot simultaneously extend types 'LegacyElementMixin' and 'HTMLElement'.
  Named property 'toggleAttribute' of types 'LegacyElementMixin' and 'HTMLElement' are not identical.`

It appears a new API for [toggleAttribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute) was introduced with a signature ` toggleAttribute(qualifiedName: string, force?: boolean): boolean;`  this is in conflict with the existing method toggleAttribute in LegacyElementMixin with a signature of `toggleAttribute(name: string, bool?: boolean, node?: Element|null): void;`.   The result is compilation errors when using Typescript 3.1 with Polymer. 

#### Live Demo
none


### Versions
Polymer 2/3 typescript defs.

--- a/lib/legacy/legacy-element-mixin.js
+++ b/lib/legacy/legacy-element-mixin.js
@@ -817,7 +817,7 @@ export const LegacyElementMixin = dedupingMixin((base) => {
      * @param {boolean=} bool Boolean to force the attribute on or off.
      *    When unspecified, the state of the attribute will be reversed.
      * @param {Element=} node Node to target.  Defaults to `this`.
-     * @return {void}
+     * @return {boolean}
      */
     toggleAttribute(name, bool, node) {
       node = /** @type {Element} */ (node || this);
@@ -826,8 +826,10 @@ export const LegacyElementMixin = dedupingMixin((base) => {
       }
       if (bool) {
         node.setAttribute(name, '');
+        return true;
       } else {
         node.removeAttribute(name);
+        return false;
       }
     }
 

--- a/lib/legacy/legacy-element-mixin.js
+++ b/lib/legacy/legacy-element-mixin.js
@@ -817,7 +817,7 @@ export const LegacyElementMixin = dedupingMixin((base) => {
      * @param {boolean=} bool Boolean to force the attribute on or off.
      *    When unspecified, the state of the attribute will be reversed.
      * @param {Element=} node Node to target.  Defaults to `this`.
-     * @return {boolean}
+     * @return {boolean} true if the attribute now exists
      */
     toggleAttribute(name, bool, node) {
       node = /** @type {Element} */ (node || this);

